REFERENCES.md

```diff
@@ -0,0 +1,56 @@
+
+
+## Table of content
+
+
+- [2CP: Decentralized Protocols to Transparently Evaluate Contributivity in Blockchain Federated Learning Environments](#2CP:-Decentralized-Protocols-to-Transparently-Evaluate-Contributivity-in-Blockchain-Federated-Learning-Environments)
+- [CAPC Learning: Confidential and Private Collaborative Learning](#CAPC-Learning:-Confidential-and-Private-Collaborative-Learning)
+- [Distributionally Robust Federated Averaging](#Distributionally-Robust-Federated-Averaging-(DRFA))
+---
+
+
+
+## 2CP: Decentralized Protocols to Transparently Evaluate Contributivity in Blockchain Federated Learning Environments
+### Contribution:
+- The paper proposes two new protocols for blockchain-based Federated Learning: **the Crowdsource Protocol** and **the Consortium Protocol**, both implemented using software framework **2CP**.
+- The two protocols use **the step-by-step evaluation** strategy to quantify the relative *contributivity* of each partner, in which the individual updates are submitted to the blockchain and later 
+**The Crowdsource protocol** supposes the existence of an *Evaluator* and *Holdout testset*
+In **the Consortium protocol** uses a new scheme, labelled as **Parallel Cross Validation**.
+  
+### Experiments and results:
+- Both protocols performed well and gave similar results on *identical and uniformly distributed datasets* as well as datasets with varying sizes
+unlike the Crowdsource protocol, the Consortium protocol struggles to give good results on datasets that had their labels flipped with various proportions 
```

Comment:
We could add the comment I made the other day at the workgroup meeting, on the fact that splitting MNIST 60k samples uniformly across multiple partners won't provide an informative scenario